Tests/BackendIntegrationTests/OfflineStoreKitIntegrationTests.swift

```diff
@@ -106,17 +106,19 @@ class OfflineStoreKit1IntegrationTests: BaseOfflineStoreKitIntegrationTests {
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     func testPurchaseWhileServerIsDownSucceedsButDoesNotFinishTransaction() async throws {
         self.serverDown()
-        try await self.purchaseMonthlyProduct()
+        try await self.purchaseMonthlyProduct(allowOfflineEntitlements: true)
 
-        self.logger.verifyMessageWasLogged(Strings.offlineEntitlements.computing_offline_customer_info, level: .info)
+        self.verifyCustomerInfoWasComputedOffline()
         self.verifyNoTransactionsWereFinished()
     }
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     func testPurchaseWhileServerIsDownPostsReceiptAfterServerComesBack() async throws {
+        self.logger.clearMessages()
```

Comment:
Small improvement in this test. The `verifyNoTransactionsWereFinished` was being affected by leftover transaction before the test started.